tests/foreman/cli/test_foremantask.py

```diff
@@ -0,0 +1,47 @@
+"""Tests for the ``foreman_tasks/api/v2/tasks`` path.
+
+:Requirement: Foremantask
+
+:CaseAutomation: Automated
```

Comment:
I don't know much about docstrings for new files, but since there's no automation in here yet shouldn't this be `NotAutomated`?

Reply:
right, I completely forgot about that!

Reply:
right, I totally forgot about that!

Reply:
@shubhamsg199 I think we should keep `Automated` here and shouldn't keep `NotAutomated` at a module level
 instead, we can add `:CaseAutomation: NotAutomated` at the test level

Reply:
Okay, keeping  `:CaseAutomation: Automated` at the modue level and adding `:CaseAutomation:` at test level